libs/common-system/src/test/java/org/codice/ddf/configuration/AbsolutePathResolverTest.java

```diff
@@ -23,42 +23,66 @@
 
 public class AbsolutePathResolverTest {
 
+  private static final AbsolutePathResolver TEST_APR =
+      new AbsolutePathResolver(Paths.get("test", "path").toString());
+
+  private static final String USER_DIRECTORY = System.getProperty("user.dir");
+
+  private static final String DEFAULT_ABSOLUTE_PATH =
+      Paths.get(USER_DIRECTORY, "test", "path").toString();
+
   @Test
   public void testNullPath() {
     System.clearProperty("ddf.home");
-    assertThat(new AbsolutePathResolver(null).getPath(), nullValue());
+    AbsolutePathResolver apr = new AbsolutePathResolver(null);
+    assertThat(apr.getPath(), nullValue());
+  }
+
+  @Test
+  public void testNullPathWithRoot() {
+    AbsolutePathResolver apr = new AbsolutePathResolver(null);
+    assertThat(apr.getPath("USER_DIRECTORY"), nullValue());
```

Comment:
❓ Did you mean to reference the variable **USER_DIRECTORY** and not a String with the value of `"USER_DIRECTORY"`?

Reply:
Yes. Yes I did.